### DISCUSSION TITLE
New architecture support for react-native

### DISCUSSION THREAD

**Discussion by tickoosunny** [react-native-elements/react-native-elements#3774] [General]
When can we expect this library to support new react-native architecture based on turbo modules?
I just wanted to know  if I migrate my app to new architecture (turbo module) based then will this library still work ? or do I need to carry out some additional work for that ?

**Comment by RyanTG** [comment]
According to https://reactnative.directory/?search=elements RNEUI has New Architecture support.

**Comment by theianmay** [comment]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋 Thank you all for your patience while our new team has been hard at work. 
We’re excited to announce that **v5.0.0-beta.1** is now available! 

This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
